.ci/wercker.sh

```diff
@@ -244,10 +244,12 @@ no-error-spring-cloud-gcp)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/spring-cloud/spring-cloud-gcp
+  checkout_from https://github.com/romani/spring-cloud-gcp
```

Comment:
this looks weird. Your fork is not going to be updated. Reason for such integration is to check latest master, isn't it?

Reply:
Upstream repository is not ready yet for execution on latest checkstyle.
Reason https://github.com/spring-cloud/spring-cloud-gcp/pull/2540
Ones it is resolved, we can start using upstream.

Reason of such validation is to use real code for testing. We can use any project , even forked, even with strict version. But ideally, yes, master HEAD.